To avoid error on when "system" download_dir is not writable for normal user.

See : http://pear.php.net/bugs/bug.php?id=11517

--- /usr/share/pear/PEAR/Downloader.php.orig	2007-07-15 12:35:38.000000000 +0200
+++ /usr/share/pear/PEAR/Downloader.php	2007-07-15 12:34:59.000000000 +0200
@@ -679,7 +679,7 @@
             return $this->_downloadDir;
         }
         $downloaddir = $this->config->get('download_dir');
-        if (empty($downloaddir)) {
+        if (empty($downloaddir) || (is_dir($downloaddir) && !is_writable($downloaddir))) {
             if (!class_exists('System')) {
                 require_once 'System.php';
             }
